samples/drivers/kscan/sample.yaml

```diff
@@ -13,3 +13,16 @@ tests:
         - "kb data(.*)"
       fixture: fixture_connect_keyboard
     depends_on: kscan
+    filter: dt_chosen_enabled("zephyr,keyboard-scan")
```

Comment:
Quick question: why filtering on chosen here and alias in tests ?

Reply:
I'm not sure why I did this-no real reason to be inconsistent I can see. I'll push an update.